safehttp/mux_test.go

```diff
@@ -315,8 +371,8 @@ func TestMuxInterceptors(t *testing.T) {
 }
 
 type setHeaderConfig struct {
-	name  string
-	value string
```

Comment:
Please remove all unrelated changes.

Reply:
What do you mean with unrelated changes? We have updated the `SetHeaderInterceptor with config` and the `SetHeaderInterceptor with mismatching config` tests to also test the commit phase. Earlier they only tested the Before phase. How is this unrelated from the PR introducing the commit phase?

I looked through all our changes and found one unrelated change in `mux_test.go` in `TestMuxInterceptors`:
```
- wantStatus: 500,
+ wantStatus: safehttp.StatusInternalServerError,
```
We have in an earlier PR replaced all numeric status codes with constants and apparently I had missed this instance. How could it not be ok to fix these kinds of tiny nits when they are discovered? The other option would be to open a new issue and PR to just change this little thing.

Reply:
Fixed!